config/locales/en.yml

```diff
@@ -333,6 +333,7 @@ en:
         meta_description: Meta Description
         front_page: Front Page
         active: Active
+        public: Public Category Content
```

Comment:
I think just "Public" or "Publicly Available" might be clearer.  Or have a radio button for "Public" or "Internal" instead of a checkbox.